dm/dm/master/openapi_controller_test.go

```diff
@@ -445,6 +447,105 @@ func (s *OpenAPIControllerSuite) TestTaskController() {
 	}
 }
 
+func (s *OpenAPIControllerSuite) TestTaskControllerWithInvalidTask() {
+	ctx, cancel := context.WithCancel(context.Background())
+	server := setupTestServer(ctx, s.T())
+	defer func() {
+		cancel()
+		server.Close()
+	}()
+
+	// create an invalid task
+	task, err := fixtures.GenNoShardErrNameOpenAPITaskForTest()
+	s.NoError(err)
+
+	// create source for task
+	{
+		// add a mock worker
+		worker1Name := "worker1"
+		worker1Addr := "172.16.10.72:8262"
+		s.NoError(server.scheduler.AddWorker(worker1Name, worker1Addr))
+		worker1 := server.scheduler.GetWorkerByName(worker1Name)
+		worker1.ToFree()
+
+		// create the corresponding mock worker
+		ctrl := gomock.NewController(s.T())
+		defer ctrl.Finish()
+		mockWorkerClient := pbmock.NewMockWorkerClient(ctrl)
+		queryResp := &pb.QueryStatusResponse{
+			Result: true,
+			Msg:    "",
+			SourceStatus: &pb.SourceStatus{
+				Source: s.testSource.SourceName,
+				Worker: worker1Name,
+			},
+			SubTaskStatus: [](*pb.SubTaskStatus){&pb.SubTaskStatus{
+				Result: &pb.ProcessResult{
+					Errors: [](*pb.ProcessError){&pb.ProcessError{
+						ErrCode:  10006,
+						ErrClass: "database",
+						ErrScope: "downstream",
+						ErrLevel: "high",
+						Message:  "fail to initialize unit Load of subtask orders_netpay_pay_inf_nuccinterconn_zh_ext_2022_[6-12] : execute statement failed: CREATE TABLE IF NOT EXISTS `dm_meta`.`orders_netpay_pay_inf_nuccinterconn_zh_ext_2022_[6-12]_lightning_checkpoint_list` (\n\t\ttask_name varchar(255) NOT NULL,\n\t\tsource_name varchar(255) NOT NULL,\n\t\tstatus varchar(10) NOT NULL DEFAULT 'init' COMMENT 'init,running,finished',\n\t\tPRIMARY KEY (task_name, source_name)\n\t);\n",
```

Comment:
I suggest to use the task name as place holder, and fill in with `task.Name` instead.